src/components/admin/event/ManageEventCard/index.tsx

```diff
@@ -17,6 +19,7 @@ interface IProps {
 const ManageEventCard = ({ event }: IProps) => {
   const router = useRouter();
   const [acmurlLoading, setAcmurlLoading] = useState(false);
+  const [discordEvent, setDiscordEvent] = useState<string | null | undefined>(event.discordEvent);
```

Comment:
Why `string | null | undefined`? Can we just have the default be null or whatever the default API response if when its not set?